guides/doc-Administering_with_Ansible/master.adoc

```diff
@@ -0,0 +1,15 @@
+:imagesdir: common/images
+:numbered:
+ifeval::["{build}" != "satellite"]
+:toc:
+:toc-placement: left
+endif::[]
+
+include::common/attributes.adoc[]
+
+:configuring-ansible:
+:context: ansible-admin
+
+= Managing {Project} with Ansible
+
+include::common/assembly_using-satellite-ansible-content-collections.adoc[]
```

Comment:
I think that creating a separate guide for this topic is overkill. The topic covers installing Ansible modules and viewing documentation for each module using the `ansible-doc` command. The topic also links to the documentation on modules for upstream - https://galaxy.ansible.com/theforeman/foreman, and downstream - https://cloud.redhat.com/ansible/automation-hub/redhat/satellite/docs. Moving this topic to the Ansible guide that you create would do the trick.

Reply:
@spetrosi  I did this because I discussed it with @evgeni and he thought it was the way forward. This section will only expand. @evgeni  mentioned that he'd hope that it would build into something like the Hammer CLI guide. 

As it currently stands, it makes little sense in the admin guide. It was buried there without any context. It is quite an important development. Foreman/Satellite & Ansible has a high uptake. 

In the proposed guide, it makes no sense as all the rest of the information is on a client-management level. 

This is not much better: https://access.redhat.com/documentation/en-us/red_hat_satellite/6.8/html-single/hammer_cheat_sheet/index

My idea, as I stated above, was to create the guide as part of this PR and then work over the next few weeks to expand it. @evgeni had already proposed some ideas on how to expand the content. This PR is already bigger than a PR should be

Let me know what you think.

Reply:
Yeah, we should sit together and talk a bit. We have lots of documentation as part of the modules and we could expose that here with a bit of scripting. Add some nice introductionary *how to use* and we should have happy uaers.

Reply:
Creating a reference guide by automatically collecting info about available modules and their parameters sounds perfect. The current Ansible Galaxy and Red Hat Ansible docs only provide a list of available modules where you can click each module and see more info. However, there is no such thing as an open reference to all modules.

Reply:
So it's up to you @spetrosi 
Do we create the guide as part of this PR, and then meet and start to fill it out (I'll work on this). 
Or do I revert for now?

Reply:
The current PR does not do the whole job of creating new guides, however, if I understand correctly, you remove the existing Ansible-related sections from the Managing and Admin guides. So if we merge this PR the sections will go until the new guides are finished and until we publish them, which can take some time. Please return the sections to the existing guides so users can still access them.
Other than that, I am fine with merging this PR as is.

Reply:
> The current PR does not do the whole job of creating new guides,

Can you please elaborate on this? 
I created the directories, they build. 
To add them to the site, they would need to build in GH actions after merging, then I could add the html link to the index of our docs.theforeman.org. 
What else needs to be done?

Reply:
I am talking about downstream more. The sections that you have deleted will disappear. However, they all are part of the new guide that I will be able to publish right away. And I see that you did not remove the section about Satellite Ansible Collection from the Admin guide just yet, so seems like we are fine with it.
OK, I will now review and merge.

Reply:
Great. I have some minor feedback from Adam Ruzicka and the #310 to apply in the next PR. 
Thank you for your review and patience. 
Evgeni is giving a talk today at 2pm your time as part of ForemanCon on Ansible Modules. I'll be taking notes for the new guide!

Reply:
> I am talking about downstream more. The sections that you have deleted will disappear. 

You could also not cherry-pick to 6.8 and they will remain for now.